### PR TITLE
[YouTube Regrets 2022] - 9263 animated timeline

### PR DESCRIPTION
**Pull request by stevedya**
Addresses #9263. This PR creates a new timeline component with parallax. Using locomotive JS. 

## Example Scrolling Video
https://user-images.githubusercontent.com/25041665/189248325-db1af89e-8fa5-439d-9fc3-cf2c8edc3178.mp4

### CODE REVIEW

**Comment by siimonevans** [comment]
@stevedya looks great, awesome work!

My only concern is the accessibility issue you've already raised on Slack (section titles overlaying images and becoming unreadable as a result). I think we should address that before proceeding, but will need to wait for Moz to respond first.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/video-thumb.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% load i18n static %}
+<div class="tw-relative video-thumb {% if size %} video-thumb--{{ size }} {% endif %} {{ class }}" data-scroll data-scroll-speed="{{ speed }}">
+  <div class="tw-flex tw-flex-col video-thumb__inner">
+    <img class="timeline__scene-image" src="{% static '_images/youtube-regrets/regrets-reporter/2022-timeline/' %}{{ image }}" alt="">
+    <p class="tw-text-[0.75rem] tw-text-sm tw-font-bold tw-mt-2">{% trans text %}</p>
```
Comment (TheoChevalier) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/video-thumb.html:5]: The `{% trans %}` tag doesn’t accept content passed dynamically, we need to localize it first in timeline.html as variables, *then* pass them through.
That being said, no need to fix this now, I’ll handle it in a future localization PR

**Review by siimonevans — APPROVED** [review]

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/video-thumb.html (hunk 1 above)
Comment (siimonevans) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/video-thumb.html:5]: Thanks @TheoChevalier - I'll resolve this for now in that case.

**Review by tbrlpld — CHANGES_REQUESTED** [review]
There are issues on mobile where the parallax is not working. There are titles overlapping with images and are unreadable. We might need to adjust the default positions there. 

Screen Shot 2022-09-09 at 07 54 42
Screen Shot 2022-09-09 at 07 55 40

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/youtube_regrets_2022.html [hunk 2]
```diff
@@ -14,30 +14,39 @@
 {% block header_wrapped %}{% endblock %}
 
 {% block content_wrapped %}
-  <main>
+  <main role="main" class="scroll-container" data-scroll-container>
```
[the remark this answers is not in the document]
Comment (siimonevans) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/youtube_regrets_2022.html:17]: I agree and would normally remove this, but it looks like this practice is being used [elsewhere](https://github.com/mozilla/foundation.mozilla.org/blob/main/network-api/networkapi/templates/pages/base.html#L101) on the site so perhaps we should leave it? I'm happy either way.

[the remark this answers is not in the document]
Comment (tbrlpld) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/youtube_regrets_2022.html:17]: Just a nitpick. We can leave that as is. I am more concerned with the overlaps.

[the remark this answers is not in the document]
Comment (stevedya) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/youtube_regrets_2022.html]: oops good catch haha

**Comment by tbrlpld** [comment]
@stevedya Not sure why, but the changes have somehow messed up the tablet hero: 
image

The footer is back though 🥳

**Review by tbrlpld — APPROVED** [review]

**Pull request merged**